testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -184,6 +185,61 @@ else if (logicalName.startsWith(WORKER_NTH)) {
         enableTrinoJavaDebugger(dockerContainer, logicalName, debugPort);
     }
 
+    private static void enableTrinoJmxRmi(DockerContainer dockerContainer)
+    {
+        String logicalName = dockerContainer.getLogicalName();
+
+        int jmxPort;
+        if (logicalName.equals(COORDINATOR)) {
+            jmxPort = 6005;
+        }
+        else if (logicalName.equals(WORKER)) {
+            jmxPort = 6009;
+        }
+        else if (logicalName.startsWith(WORKER_NTH)) {
+            int workerNumber = parseInt(logicalName.substring(WORKER_NTH.length()));
+            jmxPort = 6008 + workerNumber;
+        }
+        else {
+            throw new IllegalStateException("Cannot enable Java debugger for: " + logicalName);
+        }
+
+        enableTrinoJmxRmi(dockerContainer, logicalName, jmxPort);
+    }
+
+    private static void enableTrinoJmxRmi(DockerContainer container, String containerName, int jmxPort)
```

Comment:
Why are new methods between
`enableTrinoJavaDebugger` and  
`enableTrinoJavaDebugger` (overloads)?

can you keep them together?